### ISSUE TITLE
Is it possible to add this effect?

### ISSUE THREAD

**Issue by jordanadania** [jasoncoon/esp8266-fastled-webserver#99]
https://gist.github.com/StefanPetrick/5e853bea959e738bc6c2c2026683e3a4

I read that delay's cause problems, but this code doesn't seem to have any

In theory, would it be easy to port this into your webserver? If you say yes, I will attempt it; otherwise, I would rather save myself the time.

Thank you!

**Comment by jasoncoon** [comment]
Yeah, definitely should be possible.  I don't see any reason it should work easily.  Let me know what you find.  Good luck!
